integration/js/steps/WaitForStartMeetingReadinessCheckerButtonToBeEnabled.js

```diff
@@ -12,15 +12,15 @@ class WaitForStartMeetingReadinessCheckerButtonToBeEnabled extends AsyncAppWaitT
   }
 
   stepDescription() {
-    return 'Waiting for meeting to be created and start button to be ready';
+    return 'Waiting for logger and device controller to be initialized and start button to be ready';
   }
 
   async waitCompleteCondition() {
```

Comment:
Should this one also have timeoutThresholdInMilliSeconds? The default is 30s which is a bit long.